lang/node-mozilla-iot-gateway/files/mozilla-iot-gateway.init

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh /etc/rc.common
+# Copyright (C) 2006-2011 OpenWrt.org
```

Comment:
no, definitely not. either drop it, or use present year and your own name/org that you have rights to assign copyright too

Reply:
will remove for now